client.go

```diff
@@ -299,9 +301,9 @@ func (b *azureSecretBackend) getClientSettings(ctx context.Context, config *azur
 
 // retry will repeatedly call f until one of:
 //
-//   * f returns true
-//   * the context is cancelled
-//   * 80 seconds elapses. Vault's default request timeout is 90s; we want to expire before then.
+//   - f returns true
```

Comment:
nit: these `fmt` changes should be fixed along with some dependency updates, might be worth updating with changes from `main` 😄

Reply:
Merged in changes from main.